json-definitions/v3/tech-record/put/car/complete/index.json

```diff
@@ -71,6 +72,16 @@
         "type": "string"
       }
     },
+    "techRecord_vehicleConfiguration": {
```

Comment:
I'm confused... how does the API save this value if it's not on the json schema? (seeing as it strips all unknown values)

Reply:
it doesn't save the value, the car config is defaulted to 'other' on frontend and we always see that, even if you change it

Reply:
Ah. Yeah that's not good. Kinda feels like a bug that would need bringing into the release.

Reply:
I am not convinced vehicle config defaulted to other is a required field, seems like something we carried over from bad data